src/test/java/org/apache/commons/io/channels/ByteArraySeekableByteChannelCompressTest.java

```diff
@@ -42,6 +42,10 @@ class ByteArraySeekableByteChannelCompressTest {
 
     private final byte[] testData = "Some data".getBytes(StandardCharsets.UTF_8);
 
+    private byte[] getTestData() {
```

Comment:
This would be better named `cloneTestData` IMO to make clear to tests what kind of data we are dealing with.